### PR TITLE
Suppress a warning with -connect when no argument are provided

### PR DESCRIPTION
**Pull request by necessarily-equal**
Some context: the flatpack uses a patch to suppress the warning, this commit upstreams the patch.

Slipher seems to be ok with this patch: https://github.com/Unvanquished/updater/issues/95#issuecomment-2223449484.

This patch is currently [added manually in the flatpack](https://github.com/flathub/net.unvanquished.Unvanquished/blob/0217843/0001-Suppress-a-warning.patch), and can be dropped after this is merged.

### CODE REVIEW

**Comment by DolceTriade** [comment]
I don't understand how this suppresses the warning? Is the default level here warn?

**Comment by necessarily-equal** [comment]
Well, it doesn't make the message disappear, but at least it doesn't make it look like a bug

**Comment by DolceTriade** [comment]
I see. I would rephrase the commit message to say "demote -connect with no server to notice instead of warning" or something. LGTM

**Review by DolceTriade — APPROVED** [review]

**Comment by slipher** [comment]
LGTM
